internal/presenters/presenter_local_finding.go

```diff
@@ -0,0 +1,184 @@
+		}
+	}
+	findings_model.Findings = filtered_findings
+
+	return nil
+}
+
+func LocalFindingPresenter(doc local_models.LocalFinding, scanned_path string, org string) *LocalFindingPresentation {
+	return &LocalFindingPresentation{
+		Input:        doc,
+		ScannedPath:  scanned_path,
+		Organization: org,
+	}
+}
+
+//go:embed templates/*
+var embeddedFiles embed.FS
+
+func LoadTemplates(files []string, tmpl *template.Template) error {
+	for _, file := range files {
+		data, err := embeddedFiles.ReadFile(file)
+		if err != nil {
+			return err
+		}
+		tmpl, err = tmpl.Parse(string(data))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (p *LocalFindingPresentation) Render() (string, error) {
+	local_findings_template, err := template.New("local_finding").Parse("")
+	if err != nil {
+		return "", err
+	}
+	AddTemplateFuncs(local_findings_template)
+	err = LoadTemplates([]string{
+		TemplatePaths.LocalFindingTemplate,
+		TemplatePaths.FindingComponentTemplate,
+	}, local_findings_template)
+	if err != nil {
+		return "", err
+	}
+
+	// filter findings
+	err = FilterBySeverityThreshold(p.SeverityThreshold, &p.Input)
+	if err != nil {
+		return "", err
+	}
+
+	// TODO: Add org and scanned path to the summary
+	sum := PrepareSummary(&p.Input.Summary, p.Organization, p.ScannedPath, p.SeverityThreshold)
+
+	buf := new(bytes.Buffer)
+	main_tmpl := local_findings_template.Lookup("main")
+
+	err = main_tmpl.Execute(buf, struct {
+		Summary SummaryData
+		Results local_models.LocalFinding
+	}{
+		Summary: sum,
+		Results: p.Input,
+	})
+	if err != nil {
+		return "", err
+	}
+	return buf.String(), nil
+}
+
+func renderTemplateToString(tmpl *template.Template) func(name string, data interface{}) (string, error) {
+	return func(name string, data interface{}) (string, error) {
+		var buf bytes.Buffer
+		err := tmpl.ExecuteTemplate(&buf, name, data)
+		if err != nil {
+			return "", err
+		}
+		return buf.String(), nil
+	}
+}
+
+func valueToString(value interface{}) string {
+	if value == nil {
+		return ""
+	}
+
+	result, ok := value.(string)
+	if !ok {
+		return ""
+	}
+	return result
+}
+
+func renderWithSeverity(severity string) string {
+	var style lipgloss.TerminalColor = lipgloss.NoColor{}
+	if strings.Contains(severity, "MEDIUM") {
+		style = lipgloss.AdaptiveColor{Light: "11", Dark: "3"}
+	}
+	if strings.Contains(severity, "HIGH") {
+		style = lipgloss.AdaptiveColor{Light: "9", Dark: "1"}
+	}
+	severityStyle := lipgloss.NewStyle().Foreground(style)
+	return severityStyle.Render(severity)
+}
+
+func bold(s string) string {
+	return lipgloss.NewStyle().Bold(true).Render(s)
+}
+
+func AddTemplateFuncs(t *template.Template) {
+	var fnMap = template.FuncMap{
+		"box": func(s string) string {
+			return boxStyle.Render(s)
+		},
+		"renderToString":        renderTemplateToString(t),
+		"toUpperCase":           strings.ToUpper,
+		"renderInSeverityColor": renderWithSeverity,
+		"valueToString":         valueToString,
```

Comment:
As mentioned in standup, suggest implementing `fmt.Stringer` for this. An example with Go's "enum" types: https://go.dev/play/p/KC7Gb72b9xR

Consider also using the builtin template function `print` which wraps `fmt.Sprint`